docker_images/flair/requirements.txt

```
@@ -1,5 +1,5 @@
 starlette==0.14.2
 pydantic==1.8.1
-flair==0.11.1
+flair @ git+https://github.com/flairNLP/flair@d4ed67bf663e4066517f00397412510d90043653
 api-inference-community==0.0.23
 huggingface_hub==0.5.1
```

There seems to be an installation error as different `huggingface_hub` versions are expected for flair, so this should fix it

```suggestion
api-inference-community==0.0.25
```